audit/audit-gcp.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -x -e
+CNCF_GCP_ORG=758905017065
+
+# gcloud organizations describe $CNCF_GCP_ORG 2>&1
+# ERROR: (gcloud.organizations.describe)
+# User [hh@ii.coop] does not have permission to access organization []
```

Comment:
@thockin Can we add 'gcloud.organizations.describe' to the k8s-infra-gcp-auditors?

Reply:
That's not the permission name.  I added 'Organization Viewer' which seems to have the right permissions.  Try it and let me know?

Reply:
did you try this again?